### PR TITLE
Tests: Local debugging setup using grafana

### PR DESCRIPTION
**Pull request by bacv**
## 1. What does this PR implement?

This PR adds a docker compose file that contains only the services for debugging, without the node itself. The tests module now has a feature flag `debug`, that will configure nodes in the topology to use loki and tempo while the test is running.

This is just another tool available for debugging interactions between two or more nodes. Integration tests without the `debug` flag will work as before - write logs to file.

## 2. Does the code have enough context to be clearly understood?

Readme.md added in `tests/README.md`

## 3. Who are the specification authors and who is accountable for this PR?

@bacv

## 4. Is the specification accurate and complete?

N/A

## 5. Does the implementation introduce changes in the specification?

N/A

## Checklist

* [x] 1. Description added.
* [x] 2. Context and links to Specification document(s) added.
* [x] 3. Main contact(s) (developers and specification authors) added
* [x] 4. Implementation and Specification are 100% in sync including changes. This is critical.
* [x] 5. Link PR to a specific milestone.

### CODE REVIEW

File: tests/Cargo.toml [hunk 1]
```diff
@@ -60,3 +60,4 @@ path = "src/tests/da.rs"
 
 [features]
 metrics = ["nomos-node/metrics"]
+debug = []
```
Comment (danielSanchezQ) [inline tests/Cargo.toml:63]: Probably we should add a default feature with nothing activated? 🤔

**Review by danielSanchezQ — APPROVED** [review]

File: tests/Cargo.toml (hunk 1 above)
Comment (bacv) [reply tests/Cargo.toml:63]: Is it really necessary? Not sure what the benefit would be, but maybe it's a convention?

Comment (danielSanchezQ) [reply tests/Cargo.toml:63]: I'm not sure actually. Usually would be to activate other features by default. So probably for empty case is not necessary. Up to you.

Comment (bacv) [reply tests/Cargo.toml:63]: Lets add it when we'll need it, I'll re-request the review, minor change for clippy.

**Review by danielSanchezQ — APPROVED** [review]

**Pull request merged**